client/dashboard/components/button-group/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { __experimentalHStack as HStack } from '@wordpress/components';
+import { ReactNode } from 'react';
+import type { ComponentProps } from 'react';
+
+type ButtonGroupProps = {
+	children: ReactNode;
+};
+
+export default function ButtonGroup( {
+	children,
+	...hStackProps
+}: ButtonGroupProps & ComponentProps< typeof HStack > ) {
```

Comment:
Let's really stop people using custom spacing :)
It's already ignoring any spacing prop they pass in anywya.

```suggestion
}: ButtonGroupProps & Omit< ComponentProps< typeof HStack >, 'spacing' > ) {
```

Reply:
Good idea 👍